clifford/test/test_g3c_tools.py

```diff
@@ -29,14 +29,17 @@
 )
 
 
+RTOL_DEFAULT = 1E-4
+ATOL_DEFAULT = 1E-6
+
 @too_slow_without_jit
 class TestRotorGeneration:
     def test_generate_translation_rotor(self):
         for i in range(10000):
             euc_vector_a = random_euc_mv()
             res = generate_translation_rotor(euc_vector_a)
             res2 = (1 + ninf * euc_vector_a / 2)
-            npt.assert_almost_equal(res.value, res2.value)
+            npt.assert_allclose(res.value, res2.value, rtol=RTOL_DEFAULT, atol=ATOL_DEFAULT)
```

Comment:
You might want to use, at the top of this file,
```
assert_allclose = functions.partial(npt.assert_allclose, rtol=RTOL_DEFAULT, atol=ATOL_DEFAULT)
```
To save yourself some typing

Reply:
*functools :)